web/resources/js/user-suite.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024. The Pycroft Authors. See the AUTHORS file.
+ * This file is part of the Pycroft project and licensed under the terms of
+ * the Apache License, Version 2.0. See the LICENSE file for details
+ */
+
+function waitForKeyPress(callback: (event: KeyboardEvent) => void): void {
+  const handleKeyPress = (event: KeyboardEvent) => {
+    // Call the callback function with the event when a key is pressed
+    callback(event);
+
+    // Remove the event listener after the first key press (if you want it to happen only once)
+    // document.removeEventListener('keydown', handleKeyPress);
+  };
+
+  // Add the event listener to the document
+  document.addEventListener('keydown', handleKeyPress);
+}
+
+// Example of using the function
+waitForKeyPress((event) => {
+  console.log(`Key pressed: ${event.key}`);
+  // Add your logic here based on the key press
+    if (event.key === "ArrowRight" ){
```

Comment:
Currently, this tab selection works even if you use ←/→ in an input field (e.g. a search field). to avoid this, add a pre-check like that:
```suggestion
    if (event.target.matches("input, textarea")) { return; }
    
    if (event.key === "ArrowRight" ){    
```

Reply:
But please check if it works, I'm not 100% sure whether the CSS selector on [`matches()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches) works like that.

Reply:
it was not possible so I checked for isinstance

Reply:
But your totally right I have not thought about text areas